### ISSUE TITLE
Brewing achievement exploit

### ISSUE THREAD

**Issue by aurel85** [PyvesB/AdvancedAchievements#742]
Hi,

Actually, for brewing achievement place a potion or splash potion in a alambic, removing it and place it back (and so on) advances achievement at a very fast.
Finally, there is to make 3 potions and to output/input them back until the achievement is finished.
Furthermore, splash potions count more than classic potions.

I use the latest version of the plugin (6.1.0) and Paper 1.15.2

Thank you in advance.

**Comment by PyvesB** [comment]
Hello @aurel85,

See #740, #715, #559, #276, #239, #125 and probably others.

As a summary:
* not an easy problem to solve due to Bukkit API limitations.
* someone tried contributing a fix in #715, but that would have completely broken the category.
* I've outlined a more complex solution in https://github.com/PyvesB/AdvancedAchievements/pull/715#issuecomment-619130766. No one has yet shared thoughts; taken action or contributed a revised fix.
* on several occasions I've suggested opening a feature request on the Spigot issue tracker to improve the Bukkit API. I don't think anyone has ever done so, people simply don't seem to care enough.
* enabling the cooldown for this category is a workaround to this issue, though not perfect.

I'm wondering whether it's not worth deleting this category completely, though I'm also pretty sure some servers happily use it with the cooldown enabled. :)
